document/core/exec/instructions.rst

```diff
@@ -2999,7 +3008,7 @@ Entering :math:`\instr` with exception handler :math:`H`
 
```

Comment:
I believe the `\instr` on line 3002 needs to be `\instr^\ast`, too.

Reply:
Done, together with the suggestions from the other review comment on `exec-try-catch`.